crates/router/src/connector/payu/transformers.rs

```diff
@@ -22,6 +22,7 @@ pub struct PayuPaymentsRequest {
     description: String,
     pay_methods: PayuPaymentMethod,
     continue_url: Option<String>,
+    merchant_order_id: String,
```

Comment:
There is no field called `merchant_order_id` in payU payment request according to[ payu docs ](https://developers.payu.com/en/restapi.html#update_notification_for_order_status) . You would have to rename this `merchant_order_id` to `ext_order_id` , Could you please make necessary changes ?